shipyard-controller/main.go

```diff
@@ -214,8 +215,11 @@ func main() {
 	logController := controller.NewLogController(logHandler)
 	logController.Inject(apiV1)
 
-	sequenceMigrator := handler.NewSequenceMigrator(createEventsRepo(), createStateRepo(), createProjectRepo())
-	sequenceMigrator.Run()
+	projectsMigrator := migration.NewProjectMVMigrator(db.GetMongoDBConnectionInstance())
+	err = projectsMigrator.MigrateKeys()
+	if err != nil {
+		log.Fatalf("Unable to run projects migrator: %v", err)
```

Comment:
This seems a bit risky - If for some reason the `lastEventTypes` property  a project could not be migrated, this would mean that Keptn can not be used at all